libs/components/modals/src/lib/modules/modal/modal-footer.component.ts

```diff
@@ -9,4 +9,11 @@ import { Component, ViewEncapsulation } from '@angular/core';
   styleUrls: ['./modal-footer.component.scss'],
   encapsulation: ViewEncapsulation.None,
 })
-export class SkyModalFooterComponent {}
+export class SkyModalFooterComponent {
+  /**
+   * A collection of error messages to be displayed to the user in the event of a form error.
+   * Values assigned here will be displayed following [SKY UX style guidelines](https://developer.blackbaud.com/skyux/design/guidelines/form-design#validation-and-error-handling).
```

Comment:
I don't think you need the second sentence since it should be implied that if it's a first-class implementation it will follow SKY UX guidelines. 

The first sentence can be trimmed down a bit (describing "what" instead of "why"): "A list of error messages to be shown above the modal footer buttons."

Reply:
updated